src/github3/repos/repo.py

```diff
@@ -1341,7 +1341,7 @@ def create_tag(
             return self.create_ref("refs/tags/" + tag, sha)
 
         json = None
-        if tag and message and sha and obj_type and len(tagger) == 3:
+        if tag and message and sha and obj_type and len(tagger) == 2:
```

Comment:
So if date is present you want to fail this? It should be `>=` right?

Reply:
Good point yes ! Not sure why I did that !